package.json

```diff
@@ -60,7 +60,7 @@
     "slugify": "^1.4.6",
     "stackdriver-errors-js": "^0.8.0",
     "web3": "^1.3.0",
-    "web3modal": "^1.9.1"
+    "web3modal-dynamic-import": "1.10.1"
```

Comment:
Can you transfer `web3modal-dynamic-import` under @ensdomains org?

Reply:
sure